### PR TITLE
[O2B-1186] Remove Id input from QC flag creation page

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [ ] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- Removed Id input from QC flag creation page

Notable changes for developers:
- NA

Changes made to the database:
- NA

### CODE REVIEW

File: lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js [hunk 1]
```diff
@@ -83,12 +75,12 @@ const qcFlagTypeCreationComponent = (qcFlagTypeCreationModel) => {
...
         h(PanelComponent, [
             h(LabelPanelHeaderComponent, { for: 'color' }, 'Color'),
-            h('label.f5.mb2', 'Please fill in a hexadecimal color after the #'),
+            h('label.f5.mb2', 'Select color'),
             colorInputComponent(color, (color) => qcFlagTypeCreationModel.patchFormData({ color })),
         ]),
     ];
 
-    return h('.w-25', panel);
+    return h('.w-l', panel);
```
Comment (martinboulais) [inline lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js:83]: Another important thing is to add flex gap (using flex-column I suppose)

File: lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js [hunk 2]
```diff
@@ -83,12 +75,12 @@ const qcFlagTypeCreationComponent = (qcFlagTypeCreationModel) => {
 
         h(PanelComponent, [
             h(LabelPanelHeaderComponent, { for: 'color' }, 'Color'),
-            h('label.f5.mb2', 'Please fill in a hexadecimal color after the #'),
+            h('label.f5.mb2', 'Select color'),
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js]: applied

File: lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js [hunk 3]
```diff
@@ -24,19 +24,9 @@ import { LabelPanelHeaderComponent } from '../../../components/common/panel/Labe
  * @return {Component} Return the view of the inputs
  */
 const qcFlagTypeCreationComponent = (qcFlagTypeCreationModel) => {
-    const { id, name, method, color, bad } = qcFlagTypeCreationModel.formData;
+    const { name, method, color, bad } = qcFlagTypeCreationModel.formData;
 
     const panel = [
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js:29]: done

File: lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js (hunk 1 above)
Comment (xsalonx) [reply lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js:82]: added

[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/views/QcFlagTypes/Create/QcFlagTypeCreationPage.js]: applied

**Review by martinboulais — APPROVED** [review]

**Pull request merged**